packages/infra/lib/api-stack.ts

```diff
@@ -264,29 +288,26 @@ export class APIStack extends Stack {
     );
 
     //-------------------------------------------
-    // S3 bucket for Medical Document Uploads
+    // General lambdas
     //-------------------------------------------
-    const medicalDocumentsUploadBucket = new s3.Bucket(this, "APIMedicalDocumentsUploadBucket", {
-      bucketName: props.config.medicalDocumentsUploadBucketName,
-      publicReadAccess: false,
-      encryption: s3.BucketEncryption.S3_MANAGED,
+    const {
+      lambdaLayers,
+      cdaToVisualizationLambda,
+      documentDownloaderLambda,
+      outboundPatientDiscoveryLambda,
+      outboundDocumentQueryLambda,
+      outboundDocumentRetrievalLambda,
+    } = new LambdasNestedStack(this, "LambdasNestedStack", {
+      config: props.config,
+      vpc: this.vpc,
+      dbCluster,
+      dbCredsSecret,
+      secrets,
+      medicalDocumentsBucket,
+      sandboxSeedDataBucket,
+      alarmAction: slackNotification?.alarmAction,
     });
 
-    //-------------------------------------------
-    // FHIR Converter Service
-    //-------------------------------------------
-    let fhirConverter: ReturnType<typeof createFHIRConverterService> | undefined;
-    if (!isSandbox(props.config)) {
```

Comment:
moved within same function

Reply:
- They're for lambdas, so it makes sense to be created in the lambda stack
   - those other lambdas left in the api-stack ideally would be moved to the lambda stack as well, just didn't have time to refactor that much now
- Its another group of resources to count towards the maximum